docs/DeveloperGuide.md

```diff
@@ -206,6 +218,18 @@ The edit applicant feature allows users to edit the details of an applicant.
 4. If the check is successful, the `EditCommand#execute()` checks if the identity of the applicant after the edit is the same as the identity of another existing applicant. 
 5. If the identity is not the same, the `Model#setApplicant()` updates the details of the applicant while the `Model#updateFilteredApplicantList()` updates applicant list to display the updated applicant list.
 
+#### Design considerations
+
+##### Aspect: Syntax for editing applicants
+
+- Alternative 1 (current choice): At least one field to edit has to be provided.
+    - Pros: More convenient for user to edit an applicant's details and shorter command.
```

Comment:
You can add the pro of not accidentally editing a field that is not meant to be edited